skimage/util/_regular_grid.py

```diff
@@ -68,7 +68,7 @@ def regular_grid(ar_shape, n_points):
     stepsizes = np.round(stepsizes).astype(int)
     slices = [slice(start, None, step) for
               start, step in zip(starts, stepsizes)]
-    slices = [slices[i] for i in unsort_dim_idxs]
+    slices = tuple([slices[i] for i in unsort_dim_idxs])
```

Comment:
Why not `tuple(slices[i] for i in unsort_dim_idxs)`?

Reply:
You know, I thought you needed the square brackets to make it a list comprehension. I guess you don't :D

Reply:
@hmaarrfk I hape a PEP for you :) - https://www.python.org/dev/peps/pep-0289/

Reply:
nice one! Yeah, I still don't know generator expressions.